src/OrbitGl/TrackHeader.cpp

```diff
@@ -42,53 +42,24 @@ void TrackHeader::DoDraw(PrimitiveAssembler& primitive_assembler, TextRenderer&
 
   const float x0 = GetPos()[0];
   const float y0 = GetPos()[1];
-  const float track_z = GlCanvas::kZValueTrack;
+  const float track_z = GlCanvas::kZValueTrackHeader;
   const float text_z = GlCanvas::kZValueTrackText;
 
   // Draw tab background
-  const float label_height = GetHeight();
-  const float half_label_height = 0.5f * label_height;
+  const float label_height = GetParent()->GetHeight();
```

Comment:
Also here should probably not use the parent's height.

Reply:
done.